01.fizzbuzz/fizzbuzz.rb

```diff
@@ -1,11 +1,13 @@
-for i in 1..20
-    if i % 3 == 0 && i % 5 == 0
-        puts "FizzBuzz"
-    elsif i % 3 == 0
-        puts "Fizz"
-    elsif i % 5 == 0
-        puts "Buzz"
-    else
-        puts i
-    end
+# frozen_string_literal: true
+
+(1..20).each do |i|
+  if (i % 3).zero? && (i % 5).zero?
```

Comment:
たぶんRuboCopの自動修正でzero?を使ったんだと思いますが、個人的には

```suggestion
  if i % 3 == 0 && i % 5 == 0
```

でも別にええやん、って思ってますねー。（好みの問題）

Reply:
レビューありがとうございます！
僕もi % 3 == 0の方が分かりやすいと思いつつ別の課題で == nil は .nil? がいいという指摘があったのでRubyは比較の時 .zero? 使うのがそれっぽいのかな勘違いしました！
i % 3 == 0の方が好きなのでこれで行こうと思います！

Reply:
> Rubyは比較の時 .zero? 使うのがそれっぽいのかな勘違いしました！

そういう流派の人も中にはいると思います。
Rubyとしての決まりはないので、実務ではプロジェクト内の方針に合わせるようにしてください〜。